client/task-list/tasks/Marketing/index.tsx

```diff
@@ -141,6 +149,12 @@ export const Marketing: React.FC = () => {
 						( extension ) => extension.slug
 					),
 				} );
+				updateOptions( {
+					woocommerce_task_list_tracked_completed_actions: [
+						...trackedCompletedActions,
+						'marketing',
```

Comment:
We should probably check if this exists inside `woocommerce_task_list_tracked_completed_actions` before adding since duplicates could occur:

<img width="604" alt="Screen Shot 2021-09-01 at 4 31 07 PM" src="https://user-images.githubusercontent.com/10561050/131741269-1345c355-2b07-42a2-ad13-cc86aecb2db8.png">

Reply:
@joshuatf  Thank you for the catch 👍 Updated in https://github.com/woocommerce/woocommerce-admin/pull/7586/commits/def1b5e0eaf847b7b09d1cfc97ae572d3ff71432